brainreg_napari/register.py

```diff
@@ -409,14 +448,77 @@ def run(
             smoothing_sigma_floating,
             histogram_n_bins_floating,
             histogram_n_bins_reference,
+            save_original_orientation,
+            brain_geometry,
         )
         worker.returned.connect(add_image_layers)
         worker.start()
 
     @widget.reset_button.changed.connect
     def restore_defaults(event=None):
         for name, value in DEFAULT_PARAMETERS.items():
-            if name != "atlas_key":
+            if name not in ["atlas_key", "brain_geometry"]:
                 getattr(widget, name).value = value
 
+    @widget.check_orientation_button.changed.connect
+    def check_orientation(event=None):
```

Comment:
What is the type of `event` here? Something along the lines of `Optional[...]`

Reply:
I have no idea, I followed what was already done. I am using another approach for Napari plugins but did not want to change the whole thing here.